eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2250,6 +2250,24 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
     context.system.eventStream.publish(NetworkFeePaid(self, remoteNodeId, channelId, tx, fee, desc))
   }
 
+  /**
+    * Returns the amount of confirmation needed to safely handle the funding transaction,
+    * we makes sure the cumulative block reward exceeds the channel size.
+    *
+    * @param fundingSatoshis funding amount of the channel
+    * @return number of confirmation needed
+    */
+  def minDepthForFunding(fundingSatoshis:Satoshi): Long = fundingSatoshis match {
+    // if this is not a wumbo channel or we're not on mainnet use the default
+    case f if f <= MAX_FUNDING || nodeParams.chainHash != Block.LivenetGenesisBlock.hash =>
```

Comment:
nit: I think the check on the `chainHash` is unnecessary, IMO we do want to be able to test the scaling logic on `regtest`/`testnet`.

Reply:
It was used to make a correct guess on the actual block reward but since we're hard-coding it we don't need the `chainHash` anymore.